docs/source/tools/export.rst

```diff
@@ -0,0 +1,164 @@
+.. Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Ledger Export
+#############
+
+Export is currently an :doc:`Early Access Feature in Labs status </support/status-definitions>`.
```

Comment:
I think this should be Alpha. We do want to get this into a stable feature. See https://docs.daml.com/support/status-definitions.html for the definitions. cc @bame-da

Reply:
Oh, copy pasted from the wrong place. Fixed.